<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pa jak podjadę</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabPages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tLinie">
       <attribute name="title">
        <string>Linie</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tUklady">
       <attribute name="title">
        <string>Układy przystanków</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tNavigation">
       <attribute name="title">
        <string>Nawigacja</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
